media/livestream/api/LiveStream.Samples.Tests/UpdatePoolAsyncTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xunit;
+using System.Threading.Tasks;
+
+namespace LiveStream.Samples.Tests
+{
+    [Collection(nameof(LiveStreamFixture))]
+    public class UpdatePoolAsyncTest
+    {
+        private LiveStreamFixture _fixture;
+        private readonly UpdatePoolSample _updateSample;
+
+        public UpdatePoolAsyncTest(LiveStreamFixture fixture)
+        {
+            _fixture = fixture;
+            _updateSample = new UpdatePoolSample();
+        }
+
+        // Pool updates cannot be run with the fixture since the fixture
+        // creates a channel test resource and starts it. Pool updates
+        // cannot be run if tests are run in parallel.
+        [Fact(Skip = "pool updates require all channels to be stopped")]
+        public async Task UpdatesPoolAsync()
+        {
+            // Pool updates take a long time to run, so the test updates the peered

```

Comment:
If pool updates take a long time to run, do you think is best to show in the sample how to poll sporadically instead of until completion? See an example of this on the [generated snippet](https://github.com/googleapis/google-cloud-dotnet/blob/bff18a61c86c6210a779639a1b15f9457ed3f594/apis/Google.Cloud.Video.LiveStream.V1/Google.Cloud.Video.LiveStream.V1.GeneratedSnippets/LivestreamServiceClient.UpdatePoolRequestObjectAsyncSnippet.g.cs#L54)

Reply:
Done